src/stylesheets/components/Auth/_auth-alert.scss

```diff
@@ -0,0 +1,26 @@
+.auth-alert {
+  backdrop-filter: blur(rem(54.3656px));
+  background: map-get($color-black, 'lighter-transparent');
+  border-radius: rem(12px);
+  margin-bottom: rem(24px);
+  min-height: rem(96px);
+  padding-top: rem(15px);
+  width: 100%;
+
+  &__title {
+    color: $secondary-color;
+    font-size: map-get($font-sizes, 'md');
+    font-weight: map-get($font-weight, 'bolder');
+    letter-spacing: -0.24px;
+    line-height: rem(20px);
+    margin-bottom: rem(10px);
+  }
+
+  &__info {
+    @include alert-detail();
```

Comment:
@rafayet-monon I think we can remove this `mixin` and place those properties in here as we use just only once.

Reply:
Changed in [5af4153](https://github.com/rafayet-monon/surveyor/pull/8/commits/5af4153e9bbe864d61e6d7752d48b250e74cd1fd)